pool/src/tests/pool.rs

```diff
@@ -464,8 +464,8 @@ struct TestPool<CI> {
 impl<CI: ChainIndex + 'static> TestPool<CI> {
     fn simple() -> TestPool<ChainKVStore<MemoryKeyValueDB>> {
         let (_handle, notify) = NotifyService::default().start::<&str>(None);
-        let new_tip_receiver = notify.subscribe_new_tip(TXS_POOL_SUBSCRIBER);
-        let switch_fork_receiver = notify.subscribe_switch_fork(TXS_POOL_SUBSCRIBER);
+        let new_tip_receiver = notify.subscribe_new_tip("txs_pool");
+        let switch_fork_receiver = notify.subscribe_switch_fork("txs_pool");
```

Comment:
```
#[cfg(test)]
......
const TXS_POOL_SUBSCRIBER: &str = "txs_pool";
```
declare here, but is not used

Reply:
I missed this comment, will fix it later.